src/ngamsUtils/ngamsUtils/ngasXSyncTool.py

```diff
@@ -462,9 +462,8 @@ def _get_cluster_nodes(connection, cluster_id):
     :param cluster_id: NGAS Cluster ID (string)
     :return: List of NGAS Host IDs of the nodes in the cluster (list)
     """
-    sql = "select host_id from ngas_hosts where cluster_name='{:s}'".format(cluster_id)
-    # result = connection.query(sql, ignoreEmptyRes=0)
-    result = connection.query2(sql)
+    sql = "select host_id from ngas_hosts where cluster_name='{0}'"
+    result = connection.query2(sql, args=(cluster_id))
```

Comment:
that will require an extra `,` I think to make it an actual tuple (i.e., `args=(cluster_id,)`)